rotkehlchen/api/v1/schemas.py

```diff
@@ -1581,11 +1586,29 @@ class ExchangesResourceAddSchema(Schema):
     name = fields.String(required=True)
     location = LocationField(limit_to=SUPPORTED_EXCHANGES, required=True)
     api_key = ApiKeyField(required=True)
-    api_secret = ApiSecretField(required=True)
+    api_secret = ApiSecretField(load_default=b'')
```

Comment:
the table allows the secret to be NULL. if you do this is it actually saved as NULL? Why set it to empty string instead of null?

Reply:
That's not a good fix. That's a hack. According to the DB Schema secret can be Null. Treat it as such.